joomla/joomla-cms
# [4.0] Webauthn Image missing on admin login screen

Pull Request for Issue # 29733

### Summary of Changes
move icon to block for webauth button

### Testing Instructions
Access Joomla 4 over a SSL
Go to /administrator/
notice that icon is missing
apply pr.
in termal run `npm run build:css`
notice that icon is showing in webauth block

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/95630130-eae81980-0a46-11eb-93b3-e9961de84d34.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/95630058-c429e300-0a46-11eb-8bd0-e56951c0b4a0.png)



### Documentation Changes Required

none
